hz: 10
direct_control_mode: true
input_speed: 0.0
input_yawrate: 1.0
